### PR TITLE
[Controller Manager] Set log level to avoid log spamming

### PR DESCRIPTION
**Pull request by m1093782566**
**What this PR does / why we need it**:

Set controller manager log level to avoid log spamming.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/32503)

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
@liggitt @pmorie -- can either of you take a quick look at these log levels?  nothing stands out to me as bad per-se, but want to know if we depend on certain pre-existing log levels.

**Comment by m1093782566** [comment]
I refer to:

#32462 #32461

**Comment by m1093782566** [comment]
code rebased.

**Review by derekwaynecarr — CHANGES_REQUESTED** [review]
fix the two typos, then lgtm

**Comment by m1093782566** [comment]
Typo fixed. PTAL.

File: pkg/controller/node/nodecontroller.go [hunk 1]
```diff
@@ -499,7 +499,7 @@ func (nc *NodeController) monitorNodeStatus() error {
 				NewRateLimitedTimedQueue(
 					flowcontrol.NewTokenBucketRateLimiter(nc.evictionLimiterQPS, evictionRateLimiterBurst))
 			// Init the metric for the new zone.
-			glog.Infof("Initilizing eviction metric for zone: %v", zone)
+			glog.V(1).Infof("Initilizing eviction metric for zone: %v", zone)
```
Comment (m1093782566) [inline pkg/controller/node/nodecontroller.go]: Fixed

File: pkg/controller/podautoscaler/horizontal.go [hunk 2]
```diff
@@ -341,7 +341,7 @@ func (a *HorizontalController) reconcileAutoscaler(hpa *autoscaling.HorizontalPo
 			return fmt.Errorf("failed to rescale %s: %v", reference, err)
 		}
 		a.eventRecorder.Eventf(hpa, api.EventTypeNormal, "SuccessfulRescale", "New size: %d; reason: %s", desiredReplicas, rescaleReason)
-		glog.Infof("Successfull rescale of %s, old size: %d, new size: %d, reason: %s",
+		glog.V(2).Infof("Successfull rescale of %s, old size: %d, new size: %d, reason: %s",
```
Comment (m1093782566) [inline pkg/controller/podautoscaler/horizontal.go]: Fixed

**Comment by m1093782566** [comment]
ping @derekwaynecarr

**Review by derekwaynecarr — APPROVED** [review]

**Comment by derekwaynecarr** [comment]
please rebase, then LGTM, thanks!

**Comment by m1093782566** [comment]
code rebased.

BTW, since this PR changes many files so it's easy to conflict with other PRs.

**Comment by m1093782566** [comment]
code rebased ping @derekwaynecarr for final review.

**Comment by m1093782566** [comment]
code rebased again!

**Comment by derekwaynecarr** [comment]
Sorry, this needs another rebase, if we keep hitting problems, maybe split into fewer PRs.

**Comment by m1093782566** [comment]
@derekwaynecarr code rebased, PTAL.

Thanks!

**Comment by derekwaynecarr** [comment]
one more rebase needed

**Comment by m1093782566** [comment]
code rebased. BTW, if we hit the code-need-rebase issue again, I will consider split this PR into a few small ones.

**Comment by m1093782566** [comment]
@derekwaynecarr PTAL

**Comment by derekwaynecarr** [comment]
ideally, you would squash your commits, if this gets hit with another need to rebase please do that in the future.

**Comment by m1093782566** [comment]
Okay

File: pkg/controller/node/nodecontroller.go [hunk 3]
```diff
@@ -469,7 +469,7 @@ func (nc *NodeController) monitorNodeStatus() error {
 				NewRateLimitedTimedQueue(
 					flowcontrol.NewTokenBucketRateLimiter(nc.evictionLimiterQPS, evictionRateLimiterBurst))
 			// Init the metric for the new zone.
-			glog.Infof("Initializing eviction metric for zone: %v", zone)
+			glog.V(1).Infof("Initializing eviction metric for zone: %v", zone)
```
Comment (pmorie) [inline pkg/controller/node/nodecontroller.go]: Sorry for the late review, but did you really mean 1 here? @m1093782566

Comment (m1093782566) [inline pkg/controller/node/nodecontroller.go]: Yes, but do you have any other ideas?

**Comment by dims** [comment]
@m1093782566 Please rebase this?

@derekwaynecarr Is this really needed for v1.5? (If not, can you please switch it?)

**Comment by dims** [comment]
@m1093782566 Please rebase this?

**Comment by m1093782566** [comment]
@dims Sure

**Comment by dims** [comment]
No activity for a while, does not seem super critical for 1.5. moving it to 1.6

**Pull request closed without merging**